src/include/kernel/abi.h

```diff
@@ -29,7 +29,7 @@
 
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 16
+#define SOF_ABI_MINOR 17
```

Comment:
Not sure, this could be 16 - just depends on when kernel PR is approved.

Reply:
Not sure if ABI needs bump (MINOR is now 17, would be to 18). There's the change to the user header fir.h.